### PR TITLE
Added docstrings on spotify_api.py, improved readme

### PR DESCRIPTION
**Pull request by HyungnaLim**
Added docstrings explaining each function of spotify module.
This should solve issue #32 

Also, now spotify module does not include client credentials in code.
You should set up environment variables to make it work.

### CODE REVIEW

**Comment by HyungnaLim** [comment]
added environment variable setting and formatting on README.md

**Comment by rwalk651** [comment]
@HyungnaLim API and secret keys are sensitive information. I was fine with using them for development, but they shouldn't be in the readme! Spotify, Ticketmaster, and Google will ban those accounts.

Take a look at Clara's cat flask readme for an outline.
https://github.com/claraj/cat_flask/blob/master/README.md

**Comment by HyungnaLim** [comment]
@rwalk651 Thanks for the review. I edited and tidy up readme! also there is a small change to spotify api module (it is to make it easier to test get_token function)

**Comment by HyungnaLim** [comment]
I'm done making change to this branch. hopefully.

**Review by rwalk651 — APPROVED** [review]
Looks good!

File: README.md [hunk 1]
```diff
@@ -1,19 +1,51 @@
...
+`python -m venv venv`
+2. Activate the virtual environment:
+`venv\Scripts\Activate`
+3. Install required packages:
+`pip install -r requirements.txt`
+4. Set flask app
+`set FLASK_APP=app.py`
+`set FLASK_DEBUG=1`
+5. Run the application:
+`flask run`
```
[the remark this answers is not in the document]
Comment (HyungnaLim) [reply README.md]: Nice catch, thanks!

**Pull request merged**